src/api/Api.ts

```diff
@@ -98,6 +98,8 @@ export class ApiImpl implements Api {
                 ...rawResult,
                 paths: ApiImpl.decodeResult(rawResult, completeRequest.elevation),
             }
+        } else if (response.status !== 400) {
+            throw new Error(response.statusText)
```

Comment:
The GraphHopper API returns 500 when a request times out, so when this happens we see `Internal Server Error' for example. Not sure if this is too unfriendly for users and what else we should do. Maybe 'Unexpected Error' or even 'The request timed out'? In most cases 500 should be because the request timed out and even when the reason was something else we could still show this message? And maybe log the actual error to the console?